### PR TITLE
Youmed 284

### PR DESCRIPTION
**Pull request by ntlong1594**

### CODE REVIEW

File: src/application/controllers/api/v2/Appointmentsv2.php [hunk 1]
```diff
@@ -464,4 +464,21 @@ public function getAppointmentWithServiceIntegrated($id_service_integrated) {
             exit($this->_handleException($exception));
         }
     }
+
+    public function checkAppointmentsWorkingDate(){
```
Comment (chicanh) [inline src/application/controllers/api/v2/Appointmentsv2.php]: -> getAppointmentsWorkingDate

File: src/application/config/routes.php [hunk 2]
```diff
@@ -52,6 +52,7 @@
 | used for each operation.
 |
 */
+$route['api/v2/appointments/working-date']['get'] = 'api/v2/appointmentsv2/checkAppointmentsWorkingDate';
```
Comment (chicanh) [inline src/application/config/routes.php]: working-date -> workingdate

Comment (chicanh) [inline src/application/config/routes.php]: move to v3

Comment (ntlong1594) [reply src/application/config/routes.php]: updated

Comment (ntlong1594) [reply src/application/config/routes.php]: updated

File: src/application/controllers/api/v2/Appointmentsv2.php (hunk 1 above)
Comment (ntlong1594) [reply src/application/controllers/api/v2/Appointmentsv2.php]: updated

**Review by chicanh — APPROVED** [review]

**Pull request merged**